create-snowpack-app/app-template-minimal/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "app-template-minimal",
+  "version": "1.6.12",
+  "license": "MIT",
+  "keywords": [
+    "csa-template"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "index.js",
+  "scripts": {
+    "start": "snowpack dev",
+    "build": "snowpack build",
+    "test": "echo \"Error: No test runner added yet.\" && exit 1"
```

Comment:
The word "yet" made sense for the guide, but maybe not for a template on its own.